numba/tests/test_tuples.py

```diff
@@ -772,6 +772,48 @@ def pyfunc(a):
 
         check((10, 20))
 
+    @skip_parfors_unsupported
+    def test_untraced_value_tuple(self):
```

Comment:
Please move this test to test_parfors.py.  All parfor related tests go in that file.  That file has infrastructure for running in parallel and non-parallel and comparing results which you are doing manually here.